src/test/java/uk/gov/mca/beacons/api/mappers/NoteMapperTest.java

```diff
@@ -0,0 +1,116 @@
+package uk.gov.mca.beacons.api.mappers;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import java.time.Clock;
+import java.time.LocalDateTime;
+import java.util.UUID;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import uk.gov.mca.beacons.api.domain.Note;
+import uk.gov.mca.beacons.api.domain.NoteType;
+import uk.gov.mca.beacons.api.domain.User;
+import uk.gov.mca.beacons.api.dto.NoteDTO;
+import uk.gov.mca.beacons.api.dto.NoteDTO.Attributes;
+import uk.gov.mca.beacons.api.dto.WrapperDTO;
+
+@TestInstance(Lifecycle.PER_CLASS)
```

Comment:
No need to share state between the tests here, so I would remove the annotation on the class